nexsock-web/src-ts/ui/error-display.ts

```
@@ -0,0 +1,251 @@
+ * Helper function to get or create the messages container
+ */
+function getOrCreateMessagesContainer(): HTMLElement {
+  let container = document.getElementById('messages-container');
+  if (!container) {
+    container = document.createElement('div');
+    container.id = 'messages-container';
+    container.className = 'messages';
+    
+    // Insert after the main navigation if it exists
+    const main = document.querySelector('main');
+    if (main && main.parentNode) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)